numpy/random/mtrand.pyx

```diff
@@ -3703,14 +3715,42 @@ cdef class RandomState:
         >>> x.shape
         (3, 3, 2)
 
+        Setting mode='fast' uses Cholesky decomposition which is fast but
+        not as robust as the default method. The mode='legacy' is the old
```

Comment:
Cholesky is also robust but cannot handle semidefinite matrices or numerically close to semidefinite matrices.

Reply:
> *but cannot handle semidefinite matrices or numerically close to semidefinite matrices*

I'd call that not robust - it can fail solely due to floating point precision errors in an otherwise psd matrix.

Reply:
What I meant is that `eigh` is also not robust in that sense but we are correcting it later with the `abs`. The difference is that one of them stops the other one gives wrong results. So in that context, same robustness levels. 

The issue is that a potentially stopping algo is used as the default which wasn't the case previously.